lib/src/surveyor/README.md

```diff
@@ -0,0 +1,3 @@
+This code is copied from https://github.com/pq/surveyor.
```

Comment:
I think this'd be better if either @pq or you properly published surveyor to pub, and used it as an actual dependency. We have other tools that use packages in this fashion, and they're a pain to maintain/use.

Also, instead of copying, this could be a [Git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules), but then you'd need to add some extra instructions when people want to build the package to actually recursively clone the code.

Reply:
Added TODO and started conversation with Phil.